src/ui.py

```diff
@@ -0,0 +1,38 @@
+import os
+from Board import displayBoard
+from Game import switchPlayer
+from copy import deepcopy
+
+def clear():
+    os.system('clear')
+
+def displayPlayerBoard(boards, player, hit):
+    if (player == 1 and not hit):
+        clear()
+        input("PLAYER 2, look away. PLAYER 1, press enter when ready")
+        clear()
+        displayBoard(boards["player1"]["opponent"], "PLAYER 1'S TURN")
+        # displayBoard(boards["player1"]["primary"], "PLAYER 1")
+    elif player == 2 and not hit:
+        clear()
+        input("PLAYER 1, look away. PLAYER 2, press enter when ready")
+        clear()
+        displayBoard(boards["player2"]["opponent"], "PLAYER 2'S TURN")
+
+def displayShootResult(hit, boards, currentPlayer):
```

Comment:
I think the logic of switching the player should be moved to another method. In my opinion, it's confusing to have a method named `display` which manages the game.

Reply:
I think I will move the logic to the main, where I have the hit value. I can check it and switch the player accordingly. What do you think ?